src/protocol/http/src/main/java/org/apache/jmeter/protocol/http/proxy/DefaultSamplerCreator.java

```diff
@@ -268,7 +268,7 @@ protected void computeFromPostBody(HTTPSamplerBase sampler,
                         HTTPFileArg [] files = {new HTTPFileArg(out.getPath(),"",contentType)};
                         sampler.setHTTPFiles(files);
                     } catch (IOException e) {
-                        log.warn("Could not create binary file: {}", e);
+                        log.warn("Could not create binary file", e);
```

Comment:
Do we want the complete stacktrace to be shown here?

Reply:
The last "exception" parameter should not be present in the string format message, so `("Could not create binary file", e);` is the right way to go to show the stacktrace